app/src/main/java/edu/uc/reedws/musiclink/service/PlaylistService.kt

```diff
@@ -40,6 +52,10 @@ class PlaylistService(application: Application) {
         }
     }
 
+    /**
+     * Retrieve all the playlists from the database
+     *
+     */
```

Comment:
Added an appropriate comment to indicate that this function retrieves all the playlists from the database because localPlaylistDAO calls the PlayListDTO which contains all the playlists.

Reply:
This method actually only gets the DAO instance, it doesn't perform any database operations.
... again, I know, proving your point.